packages/l10n/country-information/country-information.js

```diff
@@ -16,7 +16,7 @@ const getCountriesForLocale = (locale, cb) => {
   // The files are named like "country-data-en-json.chunk.js" after compilation
   // https://webpack.js.org/api/module-methods/#import-
   import(/* webpackChunkName: "country-data-[request]" */
-  `./data/countries/${supportedLocale}.json`)
+  `../data/countries/${supportedLocale}.json`)
```

Comment:
these "webpackChunkNames" are most likely removed by rollup. We should look into a way to preserve them.

Reply:
What if we just preserve comments?

Reply:
Good news, rollup doesn't remove them 💯